inc/Extension/PluginController.php

```diff
@@ -311,20 +310,18 @@ protected function rebuildLocal()
     protected function loadConfig()
     {
         global $config_cascade;
-        foreach (array('default', 'protected') as $type) {
+        foreach (['default', 'protected'] as $type) {
             if (array_key_exists($type, $config_cascade['plugins'])) {
                 $this->pluginCascade[$type] = $this->checkRequire($config_cascade['plugins'][$type]);
             }
         }
         $local = $config_cascade['plugins']['local'];
         $this->lastLocalConfigFile = array_pop($local);
-        $this->pluginCascade['local'] = $this->checkRequire(array($this->lastLocalConfigFile));
-        if (is_array($local)) {
-            $this->pluginCascade['default'] = array_merge(
-                $this->pluginCascade['default'],
-                $this->checkRequire($local)
-            );
-        }
+        $this->pluginCascade['local'] = $this->checkRequire([$this->lastLocalConfigFile]);
+        $this->pluginCascade['default'] = array_merge(
```

Comment:
Good question. It might think so... but I caught it being wrong before. I disabled rules when I caught them doing things wrong. This one might have slipped through. Would be a good idea to check the history why this check had been added and if we need it back.